src/py/aspen/app/views/auspice.py

```diff
@@ -37,13 +36,13 @@ def auspice_view(phylo_tree_id: int):
 
         s3_client = s3_resource.meta.client
 
-        presigned_url = (
-            s3_client.generate_presigned_url(
+        presigned_url = s3_client.generate_presigned_url(
             "get_object",
             Params={"Bucket": phylo_tree.s3_bucket, "Key": phylo_tree.s3_key},
-            ExpiresIn=3600
-            )
+            ExpiresIn=3600,
         )
 
-        return presigned_url
+        response = make_response(json.dumps({"url": presigned_url}))
```

Comment:
only real change is here, which returns JSON. Everything else is `prettier` formatting!